staging/script.ozweather-skinpatcher/resources/lib/ozweather_skinpatcher.py

```
@@ -23,14 +22,85 @@ def get_ozweather_version() -> str:
+    except (FileNotFoundError, IOError, OSError, PermissionError, ValueError) as e:
+        Logger.error("Unable to determine if skin is already patched - assuming it hasn't been patched")
+        Logger.error(e)
+
+    if not this_skin_version_already_patched:
+        patch()
+        try:
+            os.makedirs(PROFILE, exist_ok=True)
+            with open(os.path.join(PROFILE, Store.current_skin), 'w', encoding='utf-8') as f:
+                f.write(skin_version_now)
+        except (IOError, OSError) as e:
+            Logger.error(f"Failed to write patch record")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)